eventsapi/kafka_producer.py

```diff
@@ -0,0 +1,54 @@
+import boto3
+from aiokafka import AIOKafkaProducer
+from aws_schema_registry import SchemaRegistryClient
+from kafka.serializer import Serializer
+from aws_schema_registry import DataAndSchema
+from aws_schema_registry.adapter.kafka import KafkaSerializer
+from eventsapi.settings import \
+    GLUE_REGISTRY_NAME, AWS_REGION, \
+    KAFKA_BOOTSTRAP_BROKERS
+
+
+class KafkaAvroSerializer(Serializer):
+    def serialize(self, topic: str, data_and_schema: DataAndSchema):
+        data, schema = data_and_schema
+        value = schema.write(data)
+        return value
+
+
+def build_callable_serializer(serializer):
+    def _serialize_data(data_schema_topic):
+        data, schema, topic = data_schema_topic
+        return serializer.serialize(topic, (data, schema))
+    return _serialize_data
```

Comment:
Just FYI / to discuss during our check-in today: I pushed up a commit late Friday to add this and remove the `CustomKafkaProducer` I had thrown together previously. Overriding a private method is a pretty horrific hack to resort to, and this is a much simpler / cleaner approach :joy: .

Reply:
Ahh this is much nicer 😌, I see what you did here. Glad it works.